<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DHT22 Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Updt_Reading">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Readings</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text_Main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Temp_Graph">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature
Graph</string>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>320</height>
     </size>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Temp_High">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature
Thresholds(C/F)</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Temp_Low">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Humi_High">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Humi_Low">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidity
Thresholds</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Updt_Thr">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>99</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Update
Thresholds</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C_F">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>C/F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Humi_Graph">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidity
Graph</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text_C_F">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text_DB">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
